maven-extension-rollout/rollout.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Should we use `set +e` to make the script stop on the first error?

Reply:
Do you mean `set -e`? With `set +e` the script does not automatically stop.

The problem with this is that you cannot do something (like `git checkout .mvn/extensions.xml` below) and check for the exit code, since the script immediately stops.